src/containers/Tenant/Diagnostics/TenantOverview/TenantStorage/TopTables.tsx

```diff
@@ -68,11 +71,15 @@ export function TopTables({path}: TopTablesProps) {
             name: 'Path',
             sortable: false,
             render: ({row}) => (
-                <div className={b('cell-with-popover-wrapper')}>
+                <LinkToSchemaObject
+                    className={b('cell-with-popover-wrapper')}
+                    path={row.Path as string}
```

Comment:
It's better to add some check before and pass `row.Path` without type assertion. Or at least have only `String(row.Path)`
```
if (!row.Path) {
   return null;
}        
```